src/apps/CustomSockets/CustomSocketsList.js

```diff
@@ -55,13 +53,11 @@ const CustomSocketsList = React.createClass({
       return (
         <EmptyView
           iconClassName="synicon-socket-custom-socket"
-          buttonLabel="Add a Socket"
           iconColor={Colors.purple400}
           title="Syncano Sockets"
           urlLabel="Syncano Sockets"
           description="Use Syncano CLI to install a Socket from library"
           docsUrl="http://docs.syncano.io/docs"
```

Comment:
shouldn't it redirect to more specific, less generic page, like this: `http://docs.syncano.io/docs/syncano-what-are-sockets`?